### PR TITLE
Provides usb preflight checks and standardizes error messages

### PR DESCRIPTION
**Pull request by emkll**
Closes https://github.com/freedomofpress/securedrop-workstation/issues/281
Closes https://github.com/freedomofpress/securedrop-workstation/issues/264

### Test plan
Testing the changes to this branch requires a specific branch of securedrop-workstation.

- check out https://github.com/freedomofpress/securedrop-workstation/pull/305
- Build package from this branch and install in sd-export-template
- [ ] build is successful
- Copy securedrop-export-<version>.deb to sd-export-template and install the package
- create metadata.json and sd-export archive based on the new functionality: `{"device":"usb-test"}` and `{"device":"disk-test"}`
- [ ] pre-flight test works as expected based on the usb device/drive connected to the export dedicated port

### CODE REVIEW

File: securedrop_export/export.py [hunk 1]
```diff
@@ -119,6 +124,41 @@ def extract_tarball(self):
...
 
+    def check_usb_connected(self):
+        # Rely on the output of lsusb on the bus assigned to. We might need to make this variable configurable
+        # In the future and extracted from config.json
+        p = subprocess.check_output(["lsusb", "-s", PCI_BUS_ID])
+        # Empty string means a likely wrong PCI_BUS_ID
+        if p == "":
+            msg = "ERROR_USB_CHECK"
+            self.exit_gracefully(msg)
+        n_usb = len(p.rstrip().split("\n"))
```
Comment (zenmonkeykstop) [inline securedrop_export/export.py]: the usb check fails for me unless i sub in `p.decode("utf-8")` for `p` here.

File: securedrop_export/export.py [hunk 2]
```diff
@@ -18,15 +18,21 @@
 ENCRYPTED_DEVICE = "encrypted_volume"
 BRLASER_DRIVER = "/usr/share/cups/drv/brlaser.drv"
 BRLASER_PPD = "/usr/share/cups/model/br7030.ppd"
-
+PCI_BUS_ID = "002:"
```
Comment (zenmonkeykstop) [inline securedrop_export/export.py]: PCI_BUS_ID for my sd-export-usb is `003:` but this changes intermittently on my system based on attach/detach events.

Comment (emkll) [reply securedrop_export/export.py]: I have updated this script and https://github.com/freedomofpress/securedrop-workstation/pull/305 to pass the PCI_BUS_ID from dom0 to a sd-export-* specific config file. This should now work as expected.

File: securedrop_export/export.py (hunk 1 above)
Comment (emkll) [reply securedrop_export/export.py]: what version of python are you using? Getting the following error with 3.5.7:
 ```
>       n_usb = len(p.decode("utf-8").rstrip().split("\n"))
E       AttributeError: 'str' object has no attribute 'decode'
```

Comment (emkll) [reply securedrop_export/export.py]: Nevermind my commend above, this was due to the test strings not being the correct format :man_facepalming: , this is now fixed.

**Comment by zenmonkeykstop** [comment]
(Also confirmed existing `printer, printer-test, disk` targets work as expected.)

**Pull request merged**